php/libraries/IdentifierGenerator.php

```diff
@@ -84,7 +84,12 @@ protected function generateSequentialID(string $id = ''): string
     {
         // If this is the first ID ever created, return the minimum value.
         if (count($this->getExistingIDs()) < 1) {
-            return $this->minValue;
+            return str_pad(
+                strval($this->minValue),
+                $this->length,
```

Comment:
`$this->length` doesn't seem to get set anywhere in this class, can you add a check to make sure it's an integer and throw an exception before this?

Reply:
@driusan It is set in the extended class. I have tested this PR for the CRU, It is there that I discovered the bug.

Reply:
It being set in a derived class doesn't mean it's set in every derived class. Can you please add a check to throw a reasonable exception if it's not set?